docs/CHANGELOG.mdx

```diff
@@ -0,0 +1,25 @@
+# Testkube Changelog
+
+## v1.15.0 (2025-05-23)
+
+This release is available in Helm Chart v2.321.0.
+
+## What changed?
+
+This release focused on bug fixes.
```

Comment:
We need some copy here. What did this release do? Anything worth mentioning?

Reply:
We were aiming to improve stability and performance with this release.